autoload/nvimdev.vim

```diff
@@ -56,15 +56,18 @@ function! nvimdev#init(path) abort
   endif
 
   " Use with :Neomake! make
+  " make will try to build  both dependancies and nvim
+  " Ignore entering .deps directory not to mess the quickfix-dirstack
+  let s:efm_backup=&efm
+  set efm&vim
+  let s:tmp_efm='%-Gninja: Entering directory `.deps'','
+  let s:tmp_efm.= &efm
+  set efm=s:efm_backup
```

Comment:
These lines can be simplified as:

```vim
\ 'errorformat': '%-Gninja: Entering directory `.deps'',' . &errorformat
```

Then `errorformat` can be restored after the dict.  I think `set efm=s:efm_backup` would generate an error, too.